src/platform.zig

```diff
@@ -57,6 +57,8 @@ comptime {
 
     assertHasDecl(@This().Core, "outOfMemory");
 
+    assertHasDecl(@This().Core, "nativeWindowCocoa");
```

Comment:
This can't be added here, as it would not work when building for WASM. Simply remove this line.

Reply:
removed